di-ipv-orchestrator-stub/src/main/resources/data/inheritedJWTEvidences.json

```diff
@@ -0,0 +1,42 @@
+{
+  "data": [
+    {
+      "scenario": "Cohort1",
+      "label": "Users with VoT PCL200 or PCL250, with no evidence provided",
+      "payload": {
```

Comment:
Should the value of this payload be `null`? The example shows that it should be an empty array in the final VC, and the value of this payload gets set in an array when constructing the VC.